conans/client/tools/compiler_flags.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
```

Comment:
Is this mostly a tools for users in their conanfiles, or for use by build helpers? I'd say the second, so probably should go to ``conans/build`` instead.

Reply:
moved to conans.client.build